lib/utils.c

```diff
@@ -302,10 +302,10 @@ sr_file_to_string(const char *filename,
 
     lseek(fd, 0, SEEK_SET); /* No reason to fail. */
 
-    static const size_t FILE_SIZE_LIMIT = 20000000; /* ~ 20 MB */
+    static const ssize_t FILE_SIZE_LIMIT = 20000000; /* ~ 20 MB */
```

Comment:
Why?

Reply:
If I remember correctly, Clang was complaining about the signed-unsigned comparison `size > FILE_SIZE_LIMIT` below. `size` has the type `off_t` which is to be signed [per POSIX](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/sys_types.h.html).